docs/auth/autologout.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I think we want to add this to the `sidebars.json` and include this under the auth and identity to make it discoverable